search/.env.dev

```diff
@@ -5,3 +5,9 @@ IMAGE_THUMBNAILS_BUCKET=janelia-flylight-color-depth-thumbnails
 PUBLISHED_NAMES_TABLE_SUFFIX=v3.4.1
 PPPM_DATA_BUCKET=janelia-ppp-match-prod
 ALIGN_TEMPLATES_BUCKET=janelia-flylight-color-depth
+# in dev, USER_POOL_ID needs to be defined,
+# otherwise it will create a new user pool 
+# and it will use that new empty pool instead of the old one
+# this is only for DEV, because the stack was removed 
+# with the exception of the Cognito User Pool which was locked
+USER_POOL_ID=us-east-1_PeR6iaLnN
```

Comment:
When I removed the dev stack, the userpool was not removed and this is the ID of the old user pool